tests/sfbase.py

```diff
@@ -29,10 +36,11 @@ def get_type():
     def get_properties(self, original: bool = True):
         """Configuration properties required for the tap."""
         return_value = {
-            'start_date': '2020-11-23T00:00:00Z',
```

Comment:
Looks like both our PRs are modifying get_properties.  To use the new pattern we have to return the return value and we are removing the original key word argument.